Formula/nim.rb

```diff
@@ -21,19 +21,23 @@ def install
     end
     system "/bin/sh", "install.sh", prefix
 
-    system "bin/nim e install_tools.nims"
-
-    target = prefix/"nim/bin"
-    target.install "bin/nimble"
-    target.install "dist/nimble/src/nimblepkg"
-    target.install "bin/nimgrep"
-    target.install "bin/nimsuggest"
-
     bin.install_symlink prefix/"nim/bin/nim"
     bin.install_symlink prefix/"nim/bin/nim" => "nimrod"
-    bin.install_symlink prefix/"nim/bin/nimble"
-    bin.install_symlink prefix/"nim/bin/nimgrep"
-    bin.install_symlink prefix/"nim/bin/nimsuggest"
+
+    # Building tools only for non-head branch
```

Comment:
Have they been removed? If not, please work with upstream to fix this error with them rather than disabling them.